test/jetstream/api/kv_test.exs

```diff
@@ -66,6 +66,36 @@ defmodule Jetstream.API.KVTest do
     assert :ok = KV.delete_bucket(:gnat, "KEY_PUT_TEST")
   end
 
+  describe "watch/3" do
+    setup do
+      bucket = "KEY_WATCH_TEST"
+      {:ok, _} = KV.create_bucket(:gnat, bucket)
+      %{bucket: bucket}
+    end
+
+    test "detects key added and removed keys", %{bucket: bucket} do
+      test_pid = self()
+
+      {:ok, watchres} =
```

Comment:
typo, "watchers"

Reply:
Not a typo. Short for watch result.

Reply:
ok, can we put an underscore between then, it's confusing otherwise